package.json

```diff
@@ -17,6 +17,8 @@
     "liff": "./main.js"
   },
   "dependencies": {
+    "dotenv": "^6.0.0",
+    "fs": "0.0.1-security",
```

Comment:
Why is `fs` necessary?

Reply:
I thought need to write file. but it don't need it :)